### DISCUSSION TITLE
Window Explorer Lag and High CPU Usage

### DISCUSSION THREAD

**Discussion by manhtk74** [msasanmh/DNSveil#221] [Q&A]
Hi, Im new. First off, thank you for developing and maintaining this project. It's been helpful for my work, but I’ve encountered a couple of performance issues: 
- When I start the application, it consumes a significant amount of CPU resources during the initial boot-up phase
- In addition to the above, I experience noticeable lag in Windows Explorer while the application is running
I’d be happy to provide more details, logs, or even assist with testing potential fixes if needed. 
Looking forward to your insights on resolving this.

**Comment by msasanmh** [comment]
> Hi, Im new. First off, thank you for developing and maintaining this project. It's been helpful for my work, but I’ve encountered a couple of performance issues: 
> - When I start the application, it consumes a significant amount of CPU resources during the initial boot-up phase
> - In addition to the above, I experience noticeable lag in Windows Explorer while the application is running
> I’d be happy to provide more details, logs, or even assist with testing potential fixes if needed. 
> Looking forward to your insights on resolving this.

Hi, welcome. 
Go to `Settings` -> `CPU` and increase the delay.
On the new platform that I'm working on, we won't have any performance issues. Takes time, though. 

Please give me your system specifications. It helps me to do better in the future.
